sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3099,6 +3099,18 @@ object GpuOverrides extends Logging {
       (a, conf, p, r) => new ComplexTypeMergingExprMeta[Concat](a, conf, p, r) {
         override def convertToGpu(child: Seq[Expression]): GpuExpression = GpuConcat(child)
       }),
+    expr[MapConcat](
+      "Returns the union of all the given maps",
+      ExprChecks.projectOnly(TypeSig.MAP.nested(TypeSig.commonCudfTypes + TypeSig.DECIMAL_128 +
+          TypeSig.NULL),
```

Comment:
Could you file a follow on issue so that once https://github.com/rapidsai/cudf/pull/10890 is merged in we can support more types for values of a map, and look into what would be needed to support arbitrary keys?

Reply:
Tracked by https://github.com/NVIDIA/spark-rapids/issues/5559